submissions/Oleksii_Drohachov/HTML_CSS_PopUp/index.html

```diff
@@ -0,0 +1,228 @@
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/blogs.png" alt="icon logo" /></div>
+                    List of tasks
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/calendar.png" alt="icon logo" /></div>
+                    Studying plan
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/chat.png" alt="icon logo" /></div>
+                    Kottans chat
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/maps.png" alt="icon logo" /></div>
+                    Treasure map
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/mail.png" alt="icon logo" /></div>
+                    Msgs from mentor
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/files.png" alt="icon logo" /></div>
+                    Project folder
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/photos.png" alt="icon logo" /></div>
+                    Screenshots
+                  </div>
+                </div>
+              </div>
+
+              <label class="second_label" for="second">Even more icons !</label>
+              <input class="second" name="second" id="second" type="checkbox" />
+              <div class="second_wrapper">
+                <div class="menu__list__main">
+                  <div class="menu__item__main" tabindex="0">
+                    <div class="item_box">
+                      <div>
+                        <img alt="icon logo" src="icons/tanslation.png" />
+                      </div>
+                      Vocabulary
+                    </div>
+                  </div>
+                  <div class="menu__item__main" tabindex="0">
+                    <div class="item_box">
+                      <div>
+                        <img alt="icon logo" src="icons/contacts.png" />
+                      </div>
+                      CV folder
+                    </div>
+                  </div>
+                  <div class="menu__item__main" tabindex="0">
+                    <div class="item_box">
+                      <div><img alt="icon logo" src="icons/videos.png" /></div>
```

Comment:
[Read this article](https://jakearchibald.com/2022/img-aspect-ratio/) about how to handle image aspect ratio.